intake_geopandas/geopandas.py

```diff
@@ -172,10 +178,45 @@ def _open_dataset(self):
                 )
         else:
             self._dataframe = geopandas.read_parquet(
-                self.urlpath,
-                **self._geopandas_kwargs
+                self.urlpath, **self._geopandas_kwargs
             )
 
+    def _get_schema(self):
+        if self._df is None:
+            self._df = self._to_dask()
+        dtypes = {k: str(v) for k, v in self._df._meta.dtypes.items()}
+        self._schema = Schema(
+            datashape=None,
+            dtype=dtypes,
+            shape=(None, len(self._df.columns)),
+            npartitions=self._df.npartitions,
+            extra_metadata={},
+        )
+        return self._schema
+
+    def _load_metadata(self):
```

Comment:
Why overload this function if it is just calling to `super()`?

Reply:
Good question -- I think we originally had some additional functionality, then realized that the base class already had what we needed.